query/QueryManagerImpl.java

```diff
@@ -138,7 +138,7 @@ public QueryManagerImpl(GraknTransaction.Extended transactionRPC) {
     @Override
     public Stream<ConceptMap> update(GraqlUpdate query, GraknOptions options) {
         return stream(updateReq(query.toString(), options.proto()))
-                .flatMap(rp -> rp.getInsertResPart().getAnswersList().stream())
```

Comment:
This would have thrown an exception if anyone ever tried to get the answers of an Update query!

Reply:
nice haha!